src/shared_utilities/utils.jl

```diff
@@ -686,6 +680,19 @@ function landsea_mask(
         Interpolations.Flat(),
     ),
 )
+    # Assuming 1 degree is 100 km = 1e5 m, we can estimate the resolution in
```

Comment:
I am actually adding a helper function for this in a separate PR:
```
"""
    effective_horizontal_resolution(nelements, npolynomial)
Helper function which returns the effective horizontal resolution of 
a global simulation in degrees
"""
function effective_horizontal_resolution(nelements, npolynomial)
    if npolynomial == 0
        return sqrt(360 * 101 / (nelements)^2 / 6)
    else
        return sqrt(360 * 101 / (nelements * npolynomial)^2 / 6)
    end
end
```

Do you want to add this to ClimaLand.Domains.jl instead in this PR?

Reply:
I think this would work only for a global simulation, but we also have regional simulations, so I would recommend using the functions I used in this PR

Reply:
Could you add it to src/shared_utilities/Domains.jl?

Reply:
Yes

Reply:
Shouldn't such a function live in ClimaCore?

Reply:
> Shouldn't such a function live in ClimaCore?

I am not sure how to do this in full generality in ClimaCore. 

The function makes use of an assumption that is true in ClimaLand but not in ClimaCore: that we are on planet Earth (so that 90 degrees are 10e6 meters by definition).
This assumption is also used in our regional simulations, which are boxes with LatLong coordinates. The associated spaces do not contain the radius.

One way we could do this is by taking the difference between adjacent points of the `coordinate_field`, but I don't know how to do that.

Reply:
I realized there's a simple way I can do this

Reply:
I am not sure. There seems to be no easy way to determine how many elements we have along each direction of the horizontal coordinates.